src/Compilers/CSharp/Portable/Compilation/InitializerSemanticModel.cs

```diff
@@ -175,15 +175,15 @@ internal override BoundNode Bind(Binder binder, CSharpSyntaxNode node, Diagnosti
             return base.Bind(binder, node, diagnostics);
         }
 
-        private BoundEqualsValue BindEqualsValue(Binder binder, EqualsValueClauseSyntax equalsValue, DiagnosticBag diagnostics)
+        private BoundNode BindEqualsValue(Binder binder, EqualsValueClauseSyntax equalsValue, DiagnosticBag diagnostics)
```

Comment:
Probably not. Good catch. It was necessary at some interim point before I (re)introduced BoundEqualsValue as a shared base type.